ui/qt-lib/src/Ariadne/UI/Qt.hs

```diff
@@ -93,12 +93,11 @@ qtEventSubLoop :: UiEventBQueue -> (UiEvent -> IO ()) -> Int -> IO (Either Int (
 qtEventSubLoop _ _ 0 = return $ Right ()
 qtEventSubLoop eventBQueue handler depth = do
   maybeEvent <- atomically $ tryReadTBQueue eventBQueue
-  next <- case maybeEvent of
-            Nothing -> return $ Right ()
-            Just event -> handler event >> (return $ Left $ depth - 1)
   -- This method is missing from qtah. If we notice lags here, add it to qtah and uncomment
   -- QCoreApplication.processEvents
```

Comment:
The order matters here, I guess.
Please at least expand the comment, at least.

Reply:
The idea was to let Qt run some of its own stuff after we run one of our handlers and before we proceed to the next one.

Reply:
Basically, `QCoreApplication.processEvents` must be _between_ `handler event` and `return`, right?

Reply:
Yes. But now I'm confused about when haskell will actually call these two functions (handler and processEvents).